cfme/services/catalogs/service_catalogs.py

```diff
@@ -135,7 +135,7 @@ class ServiceCatalogDetails(CFMENavigateStep):
     def step(self):
         try:
             self.prerequisite_view.service_catalogs.tree.click_path("All Services",
-                self.obj.catalog.name, self.obj.name)
+                    self.obj.catalog, self.obj.name)
```

Comment:
I see what you mean, better to bypass the catalog object and then use attributes. 
Will fix this